### DISCUSSION TITLE
Uso de condicionales dentro del 'return'

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#67] [Preguntas y respuestas] [answered]
Hola! estaba viendo de diferentes maneras de retornar 1 o 0 sin tener que usar return multiples veces y sin el uso de una variable en la que almacenarlo y encontre un ejemplo de un ejercicio en el que ponian un condicional en el retorno (por ejemplo `return numero<numero_2;`) y si el condicional era verdadero este retornaba 1 y si era falso retornaba 0.  Seria valido si uso esto en algunos ejercicios?

**Comment by martinvilu** [comment, marked as answer]
Es el resultado de la comparación, verdadero es 1 (o cualquier otro número) y 0 cuando es falso.
Una variable extra no es tanto problema.
